src/types/proceedings.d.ts

```diff
@@ -17,9 +17,10 @@ type Message = {
     date: Date;
     type: RequestType | ResponseType | 'response';
     slug?: string;
-    recipient: string;
-    email: string;
+    correspondent_address: string;
+    correspondent_email: string;
     transport_medium: TransportMedium;
     subject?: string;
     content?: string;
+    sentByMe: boolean;
```

Comment:
Because I started to move everything new to camelCase and some old variables are still snake_case in other places of the code.

Reply:
Yeah okay, fair enough. I tried to stick to the existing case in objects because this looks especially silly there, but I guess it's a huge mess already anyway, so whatevs.